### DISCUSSION TITLE
Any plans to add feature for bar code 128 for fpdf2 ?

### DISCUSSION THREAD

**Discussion by papunmohanty-homer** [py-pdf/fpdf2#719] [Q&A]
Any plans to add a feature for bar code 128 for fpdf2?

**Comment by Lucas-C** [comment]
Hi @papunmohanty-homer!

No plan for the moment, but that would be nice for sure!
I think the best way would be for us to document on https://pyfpdf.github.io/fpdf2/Barcodes.html
how to combine `fpdf2` with [python-barcode](https://python-barcode.readthedocs.io/en/stable/)

**Reply by Lucas-C** [reply]
I opened https://github.com/PyFPDF/fpdf2/issues/720 to track this idea!
Would you like to give it a go @papunmohanty-homer and submit a PR? 😊

**Reply by Lucas-C** [reply]
FYI, @ssavi-ict recently added some documentation on how to generate those barcodes:
https://pyfpdf.github.io/fpdf2/Barcodes.html#code128
